### ISSUE TITLE
Seems like behavior methods can't be in a single file?

### ISSUE THREAD

**Issue by mkgn** [bigskysoftware/_hyperscript#542]
I wanted to write small reusable behaviors separately and include as needed. So I created a file and put my behaviors 

      behavior ToggleSideBarMenu()
            on click log event.target if event.target match .parent-link toggle .menu-open tell <ul/> in me toggle @style='test'
      end

      behavior Accordion()
            on click log event.target if event.target match .accordion-button toggle .collapsed tell next <div/> toggle .show
      end  

Then I include this in my main file before the hyperscript js file. console complains 

![image](https://github.com/bigskysoftware/_hyperscript/assets/8392595/4d840d84-d58d-42dc-a67b-913ccf1e6e4a)

If I break it down to two files and include it works. Shouldn't we support this use case? So that reusable behaviors can be nicely organized and included as needed?

**Comment by mkgn** [comment]
got this to work, basically if you don't have proper end statements when it comes to a bit complicated behavior things go wrong. however in documentation it's not very clear. for example, in below behavior

      behavior DropDownButton()
    on click if I match .show 
      remove .show from me
      remove .show from next <.dropdown-menu.show/>
    else 
      add .show to next <.dropdown-menu/>
      add @style='position: absolute; inset: 0px auto auto 0px; margin: 0px; transform: translate(0px, 40px)' to next <.dropdown-menu.show/>
      add .show to me end //if this "end" is not there, any behaviors below this behavior will not work, console gives errors
    end
    end

So basically play with your "end" statements.
